RELEASES.md

```diff
@@ -0,0 +1,188 @@
+# Swift for TensorFlow Release Notes
+
+## Version 0.3
+
+### Overview
+
+This is the second public release of Swift for TensorFlow, available across
+Google Colaboratory, Linux and macOS. The focus is improving overall stability
+and refining APIs.
+
+### Notebook Environments (Colab and Jupyter)
+
+* Install SwiftPM packages using `%install` directives. See [documentation in
+  README](https://github.com/google/swift-jupyter#install-directives).
+  ([swift-jupyter#45](https://github.com/google/swift-jupyter/pull/45),
+  [swift-jupyter#48](https://github.com/google/swift-jupyter/pull/48),
+  [swift-jupyter#52](https://github.com/google/swift-jupyter/pull/52))
+* `swift-jupyter` can now be installed in a Conda environment. See
+  [documentation in
+  README](https://github.com/google/swift-jupyter#option-2-using-a-swift-for-tensorflow-toolchain-and-conda).
+
+### Swift Standard Library Enhancements
+
+* `AnyDerivative` has been added, representing a type-erased derivative.
+  ([apple/swift#23521](https://github.com/apple/swift/pull/23521))
+
+### TensorFlow Library
+
+* `Tensor` now supports advanced indexing and striding APIs.
+  ([apple/swift#24684](https://github.com/apple/swift/pull/23684))
+* `Tensor`s are now pretty-printed, based on the format of NumPy.
+  ([apple/swift#23837](https://github.com/apple/swift/pull/23837))
+* TensorFlow APIs now use `Int` instead of `Int32`.
+  ([apple/swift#24012](https://github.com/apple/swift/pull/24012))
+* The `saveV2` and `restoreV2` ops are now supported.
+  ([apple/swift#23777](https://github.com/apple/swift/pull/23777))
+* The `split` and `splitV` ops are now supported.
+  ([apple/swift#24096](https://github.com/apple/swift/pull/24096),
+  [apple/swift#24120](https://github.com/apple/swift/pull/24120))
+* Experimental APIs have been added to trace functions that take and return
+  `TensorGroup`, optionally using XLA compilation.
+  ([apple/swift#23868](https://github.com/apple/swift/pull/23868))
+
+### Swift for TensorFlow Deep Learning Library
+
+* The `Layer` protocol `applied(to:)` method has been renamed to `call(_:)`.
+  `Layer`s are now “callable” like functions, e.g. `layer(input)`.
+  * Note: this is experimental functionality that is currently [being proposed
+    through Swift
+    Evolution](https://forums.swift.org/t/se-0253-static-callables/22243).
+    Expect some changes.
+* The `context` argument has been removed from `Layer`’s `applied(to:)` method.
+  It is replaced by the `ContextManager` class, which manages a stack of
```

Comment:
`ContextManager` is not a user API. `Context` is. It's worth mentioning `Context.local`, `withContext(_:_:)` and `withLearningPhase(_:_:)`.

Reply:
It would be nice to leave a note that layers like `BatchNorm` and `Dropout` will check for the thread-local `Context` to determine whether the current learning phase is training or inference. A more important note would be like "Be sure to set the learning phase to `.training` before running a training loop".

Reply:
Thanks! Updated:

<img width="694" alt="Screen Shot 2019-04-23 at 2 39 33 PM" src="https://user-images.githubusercontent.com/5590046/56617818-ab6d0980-65d5-11e9-953e-852e532f36fd.png">

Reply:
Small nit: "context learning phase" -> "contextual learning phase"

Reply:
How about capitalizing the letter after `Note: `?

Reply:
Done.